openshift/manifests/0000_30_cluster-api_04_crd.core-cluster-api.yaml

```diff
@@ -0,0 +1,8802 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.13.0
+    exclude.release.openshift.io/internal-openshift-hosted: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/feature-set: TechPreviewNoUpgrade
```

Comment:
We need all manifests to also support `CustomNoUpgrade`, will need to update the generator at some point to support that. I believe you can comma separate values in this as a list

Reply:
So _ALL_ core CAPI and CAPI providers manifests that have `release.openshift.io/feature-set: TechPreviewNoUpgrade` should also have `release.openshift.io/feature-set: CustomNoUpgrade`?

Can you confirm? If so I'll address this in a separate PR 👍

Reply:
I believe the semantic is `release.openshift.io/feature-set: CustomNoUpgrade,TechPreviewNoUpgrade` but this will need to be tested (raising a PR and triggering CI will probably be enough proof)